examples/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2016, Dennis Sitelew
+cmake_minimum_required(VERSION 3.0)
+
+if(NOT CMAKE_BUILD_TYPE)
+  message(WARNING "You shouldn't benchmark a debug build")
```

Comment:
@dpantele It's just a waning and in example code, not library itself. It doesn't force you to use release build. I don't think it will have any negative impact on the development process.